src/LuaTransformer.ts

```diff
@@ -3200,30 +3200,16 @@ export class LuaTransformer {
         }
     }
 
-    private getAbsoluteImportPath(relativePath: string): string {
-        if (relativePath.charAt(0) !== "." && this.options.baseUrl) {
-            return path.resolve(this.options.baseUrl, relativePath);
-        }
-        return path.resolve(path.dirname(this.currentSourceFile.fileName), relativePath);
-    }
-
     private getImportPath(relativePath: string): string {
-        // Calculate absolute path to import
-        const absolutePathToImport = this.getAbsoluteImportPath(relativePath);
-        if (this.options.rootDir) {
-            // Calculate path relative to project root
-            // and replace path.sep with dots (lua doesn't know paths)
-            const relativePathToRoot = this.pathToLuaRequirePath(
-                absolutePathToImport.replace(this.options.rootDir, "").slice(1)
-            );
-            return relativePathToRoot;
-        }
-
         return this.pathToLuaRequirePath(relativePath);
     }
 
     private pathToLuaRequirePath(filePath: string): string {
-        return filePath.replace(new RegExp("\\\\|\/", "g"), ".");
+        return filePath
+            .replace(".\\", "")
+            .replace("./", "")
+            .replace("/", ".")
+            .replace("\\", ".");
```

Comment:
Backslashes can be used in filenames in some POSIX environments and should not be replaced on POSIX.

Ideally filePath should be converted using nodes path lib maybe using path.sep (https://nodejs.org/api/path.html#path_path_sep) is enough,

Reply:
Can TypeScript and Lua reference a module on a POSIX system with a backslash in its name?

Reply:
Not sure, never tried it. Nevertheless it would be nicer to use a platform independent solution here.

Reply:
I didn't use `path.sep` because Windows paths can have either one of the slashes. But I have made it so non-win32 systems do not have their backslashes removed.